cli/types/tests/chainer-examples.ts

```diff
@@ -357,8 +357,6 @@ cy.get('#result').should('not.be.focused')
 cy.get('#result').should('have.focus')
 cy.get('#result').should('not.have.focus')
 
-cy.get('#result').should('be.contain', 'text')
```

Comment:
There's no `be.contain` as far as I know, so I'm not sure what this is actually meant to test.

Reply:
`be` is just a chainable getter, from our Docs https://docs.cypress.io/guides/references/assertions.html#BDD-Assertions

>These getters are also available for BDD assertions. They don’t actually do anything, but they enable you to write clear, english sentences.

So, removing this is actually removing a valid test

```suggestion
cy.get('#result').should('contain', 'text')
```